### PR TITLE
chore(pt-br): handling redirects of chrome.com

### PR DESCRIPTION
**Pull request by PassionPenguin**
1. handling redircts of `chrome.com`

### CODE REVIEW

**Comment by PassionPenguin** [comment]
> [!WARNING]
> Entries related to chrome devtools may need a thorough cleanup. Links pointing to chrome devtools have a tremendous change and such documents on google developer site are no longer available.

File: files/pt-br/learn/common_questions/tools_and_setup/what_are_browser_developer_tools/index.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Descubra mais sobre o Inspetor em diferentes navegadores:
 
 - [Inspetor de páginas do Firefox](/pt-BR/docs/Tools/Page_Inspector)
 - [IE DOM Explorer](http://msdn.microsoft.com/en-us/library/ie/dn255008%28v=vs.85%29.aspx)
-- [Chrome DOM inspector](https://developer.chrome.com/devtools/docs/dom-and-styles) (Inspetor da Opera funciona da mesma forma)
+- [Chrome DOM inspector](https://developer.chrome.com/docs/devtools/) (Inspetor da Opera funciona da mesma forma)
```
Comment (nathipg) [inline files/pt-br/learn/common_questions/tools_and_setup/what_are_browser_developer_tools/index.md:97]:
```suggestion
- [Chrome DOM inspector](https://developer.chrome.com/docs/devtools/dom) (Inspetor da Opera funciona da mesma forma)
```

File: files/pt-br/learn/common_questions/tools_and_setup/what_are_browser_developer_tools/index.md [hunk 2]
```diff
@@ -203,7 +203,7 @@ Descubra mais sobre o console JavaScript em diferentes navegadores:
 
 - [Firefox Web Console](/pt-BR/docs/Tools/Web_Console)
 - [IE JavaScript console](http://msdn.microsoft.com/en-us/library/ie/dn255006%28v=vs.85%29.aspx)
-- [Chrome JavaScript Console](https://developer.chrome.com/devtools/docs/console) (O inspetor do Ópera funciona da mesma forma)
+- [Chrome JavaScript Console](https://developer.chrome.com/docs/devtools/) (O inspetor do Ópera funciona da mesma forma)
```
Comment (nathipg) [inline files/pt-br/learn/common_questions/tools_and_setup/what_are_browser_developer_tools/index.md:206]:
```suggestion
- [Chrome JavaScript Console](https://developer.chrome.com/docs/devtools/console) (O inspetor do Ópera funciona da mesma forma)
```

File: files/pt-br/web/api/console/assert_static/index.md [hunk 3]
```diff
@@ -45,4 +45,4 @@ Ver [Outputting text to the console](/pt-BR/docs/Web/API/console#Outputting_text
 - [WHATWG Console Standard: console.assert](https://console.spec.whatwg.org/#assert-condition-data)
 - [Opera Dragonfly documentation: Console](http://www.opera.com/dragonfly/documentation/console/)
 - [MSDN: Using the F12 Tools Console to View Errors and Status](http://msdn.microsoft.com/library/gg589530)
-- [Chrome Developer Tools: Using the Console](https://developer.chrome.com/devtools/docs/console#assertions)
+- [Chrome Developer Tools: Using the Console](https://developer.chrome.com/docs/devtools/#assertions)
```
Comment (nathipg) [inline files/pt-br/web/api/console/assert_static/index.md:48]:
```suggestion
- [Chrome Developer Tools: Using the Console](https://developer.chrome.com/docs/devtools/console)
```

File: files/pt-br/web/api/console/timestamp_static/index.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ slug: Web/API/console/timeStamp_static
 
 ## Sumário
 
-Adiciona um marcador simples para as ferramentas [Timeline](https://developer.chrome.com/devtools/docs/timeline) ou [Waterfall](/pt-BR/docs/Tools/Performance/Waterfall) do navegador. Ele deixa você relacionar um ponto no seu código com os outros eventos gravados na linha do tempo, como um evento de layout ou de pintura.
+Adiciona um marcador simples para as ferramentas [Timeline](https://developer.chrome.com/docs/devtools/) ou [Waterfall](/pt-BR/docs/Tools/Performance/Waterfall) do navegador. Ele deixa você relacionar um ponto no seu código com os outros eventos gravados na linha do tempo, como um evento de layout ou de pintura.
```
Comment (nathipg) [inline files/pt-br/web/api/console/timestamp_static/index.md:10]:
```suggestion
Adiciona um marcador simples para as ferramentas [Timeline](https://developer.chrome.com/docs/devtools/performance/timeline-reference) ou [Waterfall](/pt-BR/docs/Tools/Performance/Waterfall) do navegador. Ele deixa você relacionar um ponto no seu código com os outros eventos gravados na linha do tempo, como um evento de layout ou de pintura.
```

**Review by nathipg — APPROVED** [review]
Thanks for updating this URLs!

**Pull request merged**